queries.sql

```diff
@@ -22,4 +22,76 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 /*Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*---------------------*/
+
+BEGIN;
+
+update animals set species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```

Comment:
- [ ] Instead of calling all `animals` table, you can explicitly call  `species` from it.

### sample code:
```UPDATE animals
   SET species = 'unspecified'; 

   SELECT species from animals; 
   ROLLBACK;

   SELECT species from animals; --
```